core/config.h

```diff
@@ -4,6 +4,8 @@
 #include <stdexcept>
 #include <vector>
 
+#include <toml++/toml.hpp>
```

Comment:
iwyu: Do not export this include in a header file, it is not needed here.

Reply:
I may need this in the latest (but incomplete yet) implementation. I will remove it if it is not needed in the later implementation(s).

Reply:
I may need this in the latest implementation since the added `getTOMLNode` function requires the TOML-related return type and parameter type.